SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -210,6 +210,9 @@ namespace SpatialConstants
 	const uint16 LOCATOR_PORT = 444;
 
 	const FString DEVELOPMENT_AUTH_PLAYER_ID = TEXT("Player Id");
+
+	const FString SCHEMA_DATABASE_FILE_PATH = TEXT("Spatial/SchemaDatabase");
+	const FString SCHEMA_DATABASE_ASSET_PATH = TEXT("/Game/Spatial/SchemaDatabase");
```

Comment:
Is this constant used anywhere without ".SchemaDatabase" appended to it?

Reply:
There's a single use of it without the extension in SpatialGDKEditorSchemaGenerator::SaveSchemaDatabase()

I'm not sure if there might be a better way to handle that one.